packages/graphback/src/GraphQLBackendCreator.ts

```diff
@@ -21,17 +29,22 @@ import { logger } from './utils/logger'
 // Datamigration should be the component on it's own
 // TODO rename to backend
 export class GraphQLBackendCreator {
-
   private dataLayerManager: IDataLayerResourcesManager;
   private dbContextProvider: DatabaseContextProvider;
-  private inputContext: Type[]
+  private schemaContext: ISchemaProvider;
+  private inputContext: Type[];
+  private newSchema: string;
```

Comment:
[Leaking respojnsibility] This fields should be part of the subclass. 
GraphQLBackendCreator is part of the public API so we might just move them to separate class